andela/ah-codeofduty
# #161255354 Send email verification

#### What does this PR do?
Adds an endpoint for email verification:
api/users/verify : creates a verification link and sends it to email provided by a user upon registration
Updates requirements.txt

#### Description of Task to be completed?
An Email should be sent to a user upon successful registration. User should be able to receive a verification link via email upon successful signup

#### How should this be manually tested?
```Clone the repo and checkout ft-email-verification-161255354```
```git clone https://github.com/andela/ah-codeofduty.git```
```git checkout ft-email-verification-161255354```
```setup the postgres database on the psql command line```
```CREATE DATABASE *database_name*;```
```set up the virtual environment and create a .env file from the .env.example```
```start the server and try out the endpoints:```
```to register a user: http://127.0.0.1:8000/api/users/```
```open email to check for receipt of verification email```

#### What are the relevant pivotal tracker stories?
[Email verification #161255354](https://www.pivotaltracker.com/story/show/161255354)

#### Screenshots
<img width="853" alt="screen shot 2018-11-02 at 10 26 34" src="https://user-images.githubusercontent.com/42682106/47899240-f65d5e00-de89-11e8-9455-e9e74f3b6a06.png">
<img width="1159" alt="screen shot 2018-11-02 at 10 29 30" src="https://user-images.githubusercontent.com/42682106/47899331-4805e880-de8a-11e8-8706-0284a1b0e533.png">
